Makefile

```diff
@@ -26,6 +26,11 @@ calicobuild.created: $(BUILD_FILES)
 	docker build -f build_calicoctl/Dockerfile -t calico/build .
 	touch calicobuild.created
 
+docker:
```

Comment:
*personally* i dislike filling up the root of the project with cruddy un-checked in files.  I'd rather we had a single directory reserved for these kind of downloaded files.  but as long as .gitignore is updated I'm not overly worried.